pyro/infer/autoguide/guides.py

```diff
@@ -347,6 +347,145 @@ def median(self, *args, **kwargs):
         return self(*args, **kwargs)
 
 
+class AutoNormal(AutoGuide):
+    """This is AutoNormal, but it uses the same shape constraint logic found
+    in AutoMultivariateNormal and AutoDiagonalNormal.
+    """
+    def __init__(self, model, init_loc_fn=init_to_median, prefix='auto'):
```

Comment:
We removed the `prefix` arg in 1.0. You shouldn't need it after you make the `PyroModule` changes I suggest below.

Also, could you add an `init_scale` so this has the same signature as `AutoDiagonalNormal`? That way it will be super easy for users to swap between the two :smile:

Reply:
Got it - I removed `prefix` and added `init_scale`, which looks like it works 👍